custom_ticket/templatetags/auth_extras.py

```diff
@@ -1,5 +1,7 @@
 from django import template
 from django.contrib.auth.models import Group
+from django.utils.safestring import mark_safe
+import json
```

Comment:
Minor comment: we usually import from the standard lib a the top of the imports block.